chromium_src/chrome/browser/ui/send_tab_to_self/send_tab_to_self_bubble_controller.cc

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) 2020 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#define BRAVE_FORCE_FETCH_SYNC_DEVICES \
+  (const_cast<SendTabToSelfBubbleController*>(this))->FetchDeviceInfo()
```

Comment:
Why can't we call `FetchDeviceInfo` directly? And why do we need to fetch device info every time? It is fetched when `SendTabToSelfBubbleController` constructed

Reply:
We can't call `FetchDeviceInfo` directly because the place where I want to call it is `const` method:
```
const std::vector<TargetDeviceInfo>&
SendTabToSelfBubbleController::GetValidDevices() const {
  BRAVE_FORCE_FETCH_SYNC_DEVICES;
  return valid_devices_;
}
```

> And why do we need to fetch device info every time?
The button in url bar of `Send to your devices` does read it once in constructor, so without this patch the bubble menu will be updated only in new created tab. This patch forces to update devices each time the menu is shown.